qa/L0_batcher/batcher_test.py

```diff
@@ -1934,15 +1936,16 @@ def test_max_queue_delay_only_non_default(self):
                 t.join()
             self.check_deferred_exception()
             model_name = tu.get_zero_model_name(model_base, len(shapes), dtype)
-            self.check_status(model_name, None, 12, 12, (1, 2))
+            self.check_status(model_name, None, 12, 12, (1, 2, 3, 4))
         except Exception as ex:
             self.assertTrue(False, "unexpected error {}".format(ex))
 
     def test_max_queue_delay_only_default(self):
         # Send 12 requests with batch size 1. The max_queue_delay is set
         # to default value of 0. There should be two distinct model
         # executions. The first few requests will form a first batch
-        # and the remaining requests will form the second batch.
+        # and the remaining requests will either form the second batch
+        # or more batches depending on their arrival time.
```

Comment:
Can you explain why the originally intended behavior is not consistent and needs to be relaxed here?